Makefile

```diff
@@ -7,7 +7,7 @@ CRD_OPTIONS ?= "crd:trivialVersions=true"
 all: manager
 
 # Run tests
-test: generate fmt vet manifests
+test: generate fmt checks manifests
```

Comment:
Is it possible to have ` generate fmt vet manifests lint` instead of ` generate fmt checks manifests`?

Reply:
Re-added as per discussion